README.md

```diff
@@ -35,8 +37,10 @@ tensorboard --logdir=path/to/logs
 Or, if you are building from source:
 
 ```
-bazel build tensorflow/tensorboard:tensorboard
-./bazel-bin/tensorflow/tensorboard/tensorboard --logdir=path/to/logs
+bazel build tensorboard:tensorboard
```

Comment:
How about reverse the order. Have the good stuff be first. Then the long explanatory way. For example:

```markdown
To build and run TensorBoard from source, run:

    bazel run //tensorboard -- --logdir=path/to/logs

Or alternatively, the following:

    bazel build //tensorboard:tensorboard
    ./bazel-bin/tensorboard/tensorboard --logdir=path/to/logs
```